Documentation/Signoffs/SteeringSystem.md

```diff
@@ -86,30 +49,46 @@ where C is the number of counts, CPR is the Counts per Revolution, and A is the
 
 Similarly, the accuracy of the angle is highly important to the selection of an encoder. To find the output sensitivy of the encoder, the following equation is used:
      
-     D = 360/(Resolution * 2)       (2)
+     D = 360/(Resolution)       (2)
 
 where D is the degrees of accuracy and Resolution is the how many data bits. 
 
-##### 4. Encoder Selection
+##### 2. Encoder Selection
+
+This steering wheel design will use a TRD-NA1024NW absolute rotary encoder with a resolution of 1024. The Encoder a voltage rating of 12 to 24 VDC, and a maximum current consumption of 70 mA. The TRD-NA1024NW has 10 data bits, a VDD pin, and a GND pin. This requires a microcontroller with at least 10 digital GPIO pins and that can supply the current of up to 70 mA to the rotary encoder.
 
-This steering wheel design will use a TRD-NA1024NW absolute rotary encoder with a resolution of 1024. The Encoder has a max frequency of 20 kHz, a voltage rating of 12 to 24 VDC, and a maximum current consumption of 70 mA. Given the maximum frequency of 20 kHz, the sampling rate must be at least 40 kHz to satify the Nyquist Theorem. Using equation 2 above, the degrees of accuracy for this encoder is 0.176 degrees which means it is capable of measuring the angle of the steering wheel up to 20% of a degree. The TRD-NA1024NW has 10 data bits, a VDD pin, and a GND pin. This requires a microcontroller with at least 10 digital GPIO pins and that can supply the current of up to 70 mA to the rotary encoder.
+The encoder has 1024 unique 10-bit outputs than will be read into the MCU. The output of the encoder changes on rotation of the knob. The knob of the encoder will be connected to a gear system that is mounted to the steering column of the vehicle so that the steering wheel rotation will rotate the knob. Each 10-bit output will increment or decrement a count variable in the MCU code depending on the driection of rotation. That count will be used to calculate the angle of the wheel using equation 1 above.
```

Comment:
How do you know that the micro and the encoder will communicate?

Reply:
They will communicate because the output of the encoder is simply 10-bit Grey code that can be read by the MCU by the digital pins. Additionally, the current and voltage ratings are such that the encoder will not damage the MCU.

Reply:
That does not mean that they will communicate. Bits have voltage associated and different devices operate at different voltages. You haven't considered the output voltage of the encoder as compared to the acceptable voltage levels for the inputs.

Gray code is not a communication method. It is an encoding.